packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CheckoutConfirm/selectors.ts

```diff
@@ -13,45 +13,20 @@ export const getData = (state: RootState) => {
   const withdrawLockCheckR = selectors.components.send.getWithdrawLockCheckRule(
     state
   )
-  const sddEligibleR = selectors.components.simpleBuy.getSddEligible(state)
-  const userSDDTierR = selectors.components.simpleBuy.getUserSddEligibleTier(
-    state
-  )
-  const isUserSddVerifiedR = selectors.components.simpleBuy.isUserSddVerified(
-    state
-  )
-  const cardsR = selectors.components.simpleBuy.getSBCards(state)
 
   return lift(
     (
       bankAccounts: ExtractSuccess<typeof bankAccountsR>,
       quote: ExtractSuccess<typeof quoteR>,
       sbBalances: ExtractSuccess<typeof sbBalancesR>,
       userData: ExtractSuccess<typeof userDataR>,
-      withdrawLockCheck: ExtractSuccess<typeof withdrawLockCheckR>,
-      sddEligible: ExtractSuccess<typeof sddEligibleR>,
-      userSDDTier: ExtractSuccess<typeof userSDDTierR>,
-      isUserSddVerified: ExtractSuccess<typeof isUserSddVerifiedR>,
-      cards: ExtractSuccess<typeof cardsR>
+      withdrawLockCheck: ExtractSuccess<typeof withdrawLockCheckR>
     ) => ({
       bankAccounts,
       quote,
       sbBalances,
       userData,
-      withdrawLockCheck,
-      isSddFlow: sddEligible.eligible || userSDDTier === 3,
```

Comment:
main issue was with this one, anyhow since we do not need it at the moment I removed all SDD related stuff.
In case that it's hard change we can revert this back and make sure that user tier do not be included in this check

Reply:
if that is the issue, can we just set it to `false`?

Reply:
yes it will work